target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -247,6 +247,18 @@ define Device/redmi_ax6
 endef
 TARGET_DEVICES += redmi_ax6
 
+define Device/spectrum_sax1v1k
+	$(call Device/FitImage)
```

Comment:
`$(call Device/EmmcImage)` you need to call this too

Reply:
fixed and confirmed it compiles and works fine